### DISCUSSION TITLE
Is it possible to create a floating window on load?

### DISCUSSION THREAD

**Discussion by bennyweise** [caplin/FlexLayout#368] [General]
Hi, I'm just getting started with FlexLayout, and I'm new to front-end work in general, so I may be framing this question wrong. I would like to create a layout that has a floating window shown when the page is loaded. Simply setting `floating: true` on a Tab doesn't do this, so I was wondering if there was a way this could be achieved?

**Comment by david-trumid** [comment]
make sure it is enabled in the global options, this works for me
